commands/entity.go

```diff
@@ -34,7 +35,11 @@ func (cmd *Entity) Run(scope scope.Scope, c *cli.Context) {
 	if c.Args().Present() {
 		cmd.show(scope.Application, c.Args().First())
 	} else {
-		cmd.list(scope.Application)
+		if scope.Entity == "" {
```

Comment:
Changed to test on equality of scope.Application and scope.Entity.
Also changed format of show for a single entity to be 2 columns so that it works better on standard width terminals.